collection-scripts/gather_network_logs

```diff
@@ -19,6 +19,8 @@ function gather_multus_data {
 	  oc -n openshift-multus logs ${MULTUS_POD} > "${NETWORK_LOG_PATH}"/"${MULTUS_POD}".log 2>&1 & PIDS=($!)
   done
 
+  oc get multi-networkpolicy -o yaml --all-namespaces > "${NETWORK_LOG_PATH}"/multi_networkpolicy 2>&1 & PIDS=($!)
```

Comment:
@davemulford should we split up some network collection (like this); out of the gatehr_network_logs script (because they are general vs host dependent)? 

@knobunc what is your thought on this?